pkg/tuned/controller.go

```diff
@@ -1301,6 +1387,19 @@ func RunInCluster(stopCh <-chan struct{}, version string) error {
 		panic(err.Error())
 	}
 
+	profiles, recommended, err := profilesRepackPath(tunedRecommendFile, tunedProfilesDirCustom)
+	if err != nil {
+		// keep going, immediate updates are expected to work as usual
+		// and we expect the vast majority of updates to be immediate anyway
+		klog.Errorf("error repacking the profile: %v", err)
+		klog.Infof("deferred updates likely broken")
```

Comment:
Thought about this. I'm reluctant to add new warnings because they will need to be undone when we move to structured logging, as there is not a structured `Warningf` counterpart